pages/questions/result.vue

```diff
@@ -49,7 +64,23 @@ const result = ref<HTMLDivElement | null>(null);
 const { isFormFailed } = storeToRefs(userStore);
 // const failingReason = computed(() => userStore.failingReason);
 
-// Função para voltar à tela anterior
+const failingReasons = computed(() => {
+  if (!userStore.failingReasons) return [];
+  return userStore.failingReasons.split(". ").filter(Boolean);
+});
+
+function goAgendarDoacao() {
+  window.location.href = "https://eventos.hemocione.com.br/";
+}
+
+function goIrmaoDeSangue() {
+  window.location.href = "https://apoie.hemocione.com.br/";
+}
+
+function goOndeDoar() {
+  window.location.href = "https://ondedoar.hemocione.com.br/";
```

Comment:
@hannahbgolds esses 3 links, precisamos colocar no nuxt.config.ts e usar como variavel de ambiente
outro ponto legal é usar a chamada `navigateTo` do nuxt (https://nuxt.com/docs/api/utils/navigate-to#navigating-to-an-external-url)